patterns/2-structured/innersource-portal.md

```diff
@@ -52,10 +52,33 @@ Key properties of the portal are:
 
 When launching the portal, a communications campaign promoting the addition of InnerSource data files or meta-data to code repositories should be considered, to bolster the number of projects displayed within the portal.
 
+### Implementations
+
+
+#### SAP Project Portal
+
 A [reference implementation](https://github.com/SAP/project-portal-for-innersource) of an InnerSource portal is available on GitHub and open for contributions. It lists all InnerSource projects of an organization in an interactive and easy to use way. Projects can self-register using a dedicated GitHub topic and provide additional metadata.
 
 ![Example of an InnerSource Portal](../../assets/img/portal-overview.png "Example of an InnerSource Portal")
 
+#### Wiki
+
+As a simple way to start, you can set aside a page on an internal wiki for listing out available projects.
+An easy way to display this information is in a table with columns giving just a little bit of extra information about the projects.
+Try to have just enough columns so that viewers can determine if they want to learn more about the project, but no more.
+Too much information will make the page overwhelming and difficult to use.
+
+Here is a sample set of columns:
+
+* **Name**.  Name of the project (optionally linked to its homepage).
+* **Brief Description**.
+* **Technology Pre-requisites**.  You must use these technologies in order to on-board to the project.
+* **Getting Started**.  Link to instructions on how to start using the project.
+* **Chat**.  Link to a chat channel to ask questions about the project.
+* **Sponsoring Team**.  Seeing if a team is behind the project can help others to have the confidence to use it.
```

Comment:
I have seen "Maintaining Team" and "Host Team" for this in the ISC.

Is there a particular reason why you like "Sponsoring" better here?
e.g. are you worried that the "maintaining" emphasizes a notice of mine vs yours, and with that goes somewhat counter to the core idea of InnerSource?

Reply:
Changed to `Host` team.  Yes - maintainer has some connotation that goes against parts of InnerSource.